drivers/regulator/regulator_axp192.c

```diff
@@ -64,7 +64,7 @@ static const struct linear_range dcdc1_ranges[] = {
 	LINEAR_RANGE_INIT(700000U, 25000U, 0x00U, 0x7FU),
 };
 
-static const struct regulator_axp192_desc dcdc1_desc = {
+__maybe_unused static const struct regulator_axp192_desc dcdc1_desc = {
```

Comment:
This regulator is only used in M5stick/M5Stack boards. Not all of this descriptors are used, causing compliance check errors. So make it `__maybe_unused` to overcome it.

Reply:
I'm wondering if there is a way to address these regulator stuff via device tree.

To be clear, I'm not against of the maybe_unused thing.